into/backends/sparksql.py

```diff
@@ -57,7 +65,16 @@ def rdd_to_sqlcontext(ctx, rdd, name=None, dshape=None, **kwargs):
     if isdimension(dshape.parameters[0]):
         dshape = dshape.measure
     sql_schema = dshape_to_schema(dshape)
-    sdf = ctx.applySchema(rdd, sql_schema)
+
+    try:
+        sdf = ctx.applySchema(rdd, sql_schema)
+    except TypeError:
+        # TODO: gross
+        # we have to convert a tuple of strings to have typed elements
+        # this seems like it would be horribly inefficient
+        rdd = rdd.map(lambda x: convert(np.ndarray, [x], dshape=dshape).item())
```

Comment:
We used to.  Also consider `rdd.mapPartitions` which will do this same operation but in bulk across a list of tuples at a time.